network/network.go

```diff
@@ -151,6 +151,13 @@ type network struct {
 	myIPs map[string]struct{} // set of IPs that resulted in my ID.
 	peers map[ids.ShortID]*peer
 
+	// tracking peerAliasIPs prevents the network
+	// from trying to open multiple connections
+	// to the same peer.
+	peerAliasIPs         map[string]struct{} // must hold stateLock to access/modify
```

Comment:
Can you specify what the key is and what its existence in the map means?

Reply:
I was copying the precedent set by the other IP tracking structs: https://github.com/ava-labs/avalanchego/blob/8be88a342fced5522cd503b72f49aae450eea863/network/network.go#L143-L144
https://github.com/ava-labs/avalanchego/blob/8be88a342fced5522cd503b72f49aae450eea863/network/network.go#L147

I'll add a little better description around all these to say it is the stringified IP.